Vagrantfile

```diff
@@ -0,0 +1,26 @@
+Vagrant.configure("2") do |config| 
+    config.vm.box = "hashicorp/bionic64"
+    config.vm.network "forwarded_port", guest: 5000, host: 5000
+    config.vm.provision "shell", privileged: true, inline: <<-SHELL 
+    config.vm.provision "shell", path: "scripts/setup_ssh.sh"
+        sudo apt-get update
+        sudo apt-get -y install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+        git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+        echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+        echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+        echo 'eval "$(pyenv init --path)"' >> ~/.profile
+        source ~/.profile
+        pyenv install 3.8.5
+        pyenv global 3.8.5
+        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+    SHELL
+    config.trigger.after :up do |trigger|
+        trigger.name = "Launching App"
+        trigger.info = "Running the TODO app setup script" 
+        trigger.run_remote = {privileged: false, inline: "
```

Comment:
Because poetry has been set up for the root user, this will also need to be run privileged. Without this, I see "poetry not found" errors

Reply:
I have updated so both are now true